core/field_angle.ts

```diff
@@ -404,14 +402,14 @@ export class FieldAngle extends FieldInput<number> {
    *
    * @param e Keyboard event.
    */
-  protected override onHtmlInputKeyDown_(e: Event) {
+  protected override onHtmlInputKeyDown_(e: KeyboardEvent) {
     super.onHtmlInputKeyDown_(e);
     const block = this.getSourceBlock();
     if (!block) {
       throw new UnattachedFieldError();
     }
 
-    const keyboardEvent = e as KeyboardEvent;
+    const keyboardEvent = e;
```

Comment:
You can just use `e` directly since we're not casting anymore.

Reply:
Updated!